### PR TITLE
Make the docs more readable/usable

### PR DESCRIPTION
**Pull request by mvadari**
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -45,7 +45,18 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "alabaster"
+# html_theme = "alabaster"
+
+# on_rtd is whether we are on readthedocs.org
+on_rtd = os.environ.get("READTHEDOCS", None) == "True"
```
Comment (mDuo13) [inline docs/conf.py:51]: I believe `.get(x, None)` is equivalent to `.get(x)`

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Tutorial
+========
+
+Insert tutorial stuff here
```
Comment (mDuo13) [inline docs/source/tutorial.rst:4]: Not sure it's ideal to have the tutorials duplicated in this repo and in xrpl.org. May as well omit this until we have actual content to put in it.

**Review by mDuo13 — CHANGES_REQUESTED** [review]
Mostly looks good, but I think we should not leave a tutorials placeholder here.

**Comment by mDuo13** [comment]
Ah, I missed the fact that you need to do `poetry install` to get the sphinx-rtd-theme dependency. Building the API docs failed until I did that.

**Comment by mDuo13** [comment]
The RTD theme is certainly an improvement, but on the other hand I think it makes it more evident how the actual structure of the repo is still more a jumble of parts than a coherent whole yet.

The build step reported that some pages aren't used. We should probably fix that:

```
checking consistency... xrpl-py/docs/source/modules.rst: WARNING: document isn't included in any toctree
xrpl-py/docs/source/xrpl.binarycodec.binary_wrappers.rst: WARNING: document isn't included in any toctree
xrpl-py/docs/source/xrpl.binarycodec.definitions.rst: WARNING: document isn't included in any toctree
xrpl-py/docs/source/xrpl.binarycodec.types.rst: WARNING: document isn't included in any toctree
```

**Review by ledhed2222 — APPROVED** [review]

**Review by mDuo13 — APPROVED** [review]

**Pull request merged**